### ISSUE TITLE
Get groups by requesting the memberOf property of the users

### ISSUE THREAD

**Issue by mdouchin** [larskanis/pg-ldap-sync#43]
Hi,

I must sync with an Active Directory server which does not return any `member` when requesting the groups details, but instead adds a `memberOf` property when requesting a user details.

The user entry has an 'memberOf' attribute value for each group (as opposed to the group having a 'member' attribute value for each user)

If I understood correctly, pg-ldap-sync cannot yet use this user `memberOf` property, but expects the group `member` property ?

**Comment by larskanis** [comment]
You're right - pg-ldap-sync needs the `member` property and can not (yet) work with `memberOf`. 

We're using two separate Active Directory trees and they both have the `member` property. Is it something that can be enabled?

**Comment by mdouchin** [comment]
Thanks for your answer. 
I will ask the Active Directory manager to see if the `member` property could be added. I hope it would be easy.
